react/src/components/blocks/ChatWindow/ChatHeader/ChannelMembers/ChannelMembers.js

```diff
@@ -6,20 +6,8 @@ import userIconImg from '../../../../../static/images/usericon.png'
 function ChannelMembers({ members = [] }) {
   return (
     <div className={styles.channelMembers}>
```

Comment:
@sriramanvellingiri Any specific reason for switching to a hardcoded users list over the existing dynamic user list (which changes according to channel)?

Reply:
@curioustushar the rendering logic code is updated now. checked with real stack, if the member count is 1 the image icon doesnt appear and if the count is more, only two image icon will be appearing,

Reply:
@sriramanvellingiri LGTM.